src/DIRAC/Resources/Computing/AREXComputingElement.py

```diff
@@ -665,6 +673,8 @@ def submitJob(self, executableFile, proxy, numberOfJobs=1, inputs=None, outputs=
             jobReference = self._arcIDToJobReference(arcJobID)
             batchIDList.append(jobReference)
             stampDict[jobReference] = diracStamp
+            secretDict[currentSecret] = {}
+            secretDict[currentSecret]["PilotStamps"] = [diracStamp]  # Used by DiracX to associate secrets and pilots
```

Comment:
Why not:
```suggestion
            secretDict[currentSecret]= diracStamp
```

Ok now I understand. It's indeed easier to pass a `diracXSecrets` list rather than a general `env` because you anyway need to associate the secret to the right stamp...
Unless you return another structure such as:

```python
result["PilotEnv"] = {stamp1: env1, stamp2: env2}
```

Which is heavy, but more generic (I am not sure what's best...